source/includes/localhost-connection.rst

```diff
@@ -0,0 +1,16 @@
+.. note:: Connect to localhost
+
+    If you want to connect to a database server running locally on its
+    default port, change the Atlas connection string to
+    ``"mongodb://localhost"``.
+
+    If you have a specific port you want to connect to, change the Atlas
+    connection string to ``"mongodb://localhost:<port_number>"``. 
```

Comment:
Nitpicks:

I'd like to see these converted to a more task-oriented, imperative tone. In general, 'If you want is already implied, so just "To connect to a database...".

I don't think mentioning 'Atlas' is helpful here, as an srv uri isn't unique to Atlas.

Lastly, it's safe to remove the word "server".